--- a/desmume/src/frontend/libretro/Makefile.libretro	2018-09-09 17:25:30.299207550 +0200
+++ b/desmume/src/frontend/libretro/Makefile.libretro	2018-09-09 17:21:15.835543000 +0200
@@ -75,10 +75,8 @@
          CXX = g++-x86
       endif
    else
-      LIBS := -lpthread -lGL -lpcap
-      CXXFLAGS += -DHAVE_OPENGL -std=gnu++11
-      DESMUME_OPENGL = 1
-	DESMUME_OPENGL_CORE = 1
+      LIBS := -lpthread -lpcap
+      CXXFLAGS += -std=gnu++11
    endif
 
    ifneq (,$(findstring clang,$(platform)))
