--- a/menu/menu_driver.c	2023-08-17 17:35:47.328058500 +0200
+++ b/menu/menu_driver_2.c	2023-08-17 17:42:40.014684249 +0200
@@ -7617,7 +7617,7 @@
                   entry->label, entry->type, i, entry->entry_idx);
          break;
       case MENU_ACTION_SEARCH:
-         menu_input_dialog_start_search();
+         //menu_input_dialog_start_search();
          break;
       case MENU_ACTION_SCAN:
          if (cbs && cbs->action_scan)
