--- a/gfx/drivers/oga_gfx.c
+++ b/gfx/drivers/oga_gfx.c
@@ -547,6 +547,7 @@ static bool oga_frame(void *data, const void *frame, unsigned width,
       unsigned height, uint64_t frame_count,
       unsigned pitch, const char *msg, video_frame_info_t *video_info)
 {
+   settings_t *settings        = config_get_ptr();
    oga_video_t *vid            = (oga_video_t*)data;
    oga_framebuf_t* page        = vid->pages[vid->cur_page];
    oga_surface_t *page_surface = page->surface;
@@ -581,10 +582,17 @@ static bool oga_frame(void *data, const void *frame, unsigned width,
       width        = vid->menu_surface->width;
       height       = vid->menu_surface->height;
       aspect_ratio = (float)width / height;
+      if (settings->bools.video_oga_vertical_enable) {
+       vid->rotation = 0;

+       if (settings->bools.video_ogs_vertical_enable)
+        vid->rotation = HAL_TRANSFORM_ROT_180;
+
+        // RARCH_LOG("EmuELEC Rotation set to %d", vid->rotation);
+      }
       oga_calc_bounds(&r, vid->drm_width, vid->drm_height, width, height, aspect_ratio, vid->display_ar);
       oga_blit(vid->menu_surface, 0, 0, width, height,
-            page_surface, r.y, r.x, r.h, r.w, HAL_TRANSFORM_ROT_270, vid->scale_mode, 0);
+      page_surface, r.y, r.x, r.h, r.w, vid->rotation, vid->scale_mode, 0);
    }
    else
 #endif
@@ -608,6 +616,14 @@ static bool oga_frame(void *data, const void *frame, unsigned width,
          }
       }

+   if (settings->bools.video_oga_vertical_enable) {
+    vid->rotation = HAL_TRANSFORM_ROT_270;
+
+   if (settings->bools.video_ogs_vertical_enable)
+    vid->rotation = HAL_TRANSFORM_ROT_90;
+
+       // RARCH_LOG("EmuELEC Rotation set to %d", vid->rotation);
+}
       oga_calc_bounds(&r, vid->drm_width, vid->drm_height, width, height, aspect_ratio, vid->display_ar);
       oga_blit(vid->frame_surface, 0, 0, width, height,
             page_surface, r.y, r.x, r.h, r.w, vid->rotation, vid->scale_mode, blend);
@@ -615,9 +631,21 @@ static bool oga_frame(void *data, const void *frame, unsigned width,

    if (msg)
    {
+
+    if (settings->bools.video_oga_vertical_enable) {
+     int rotation2 = 0;
+
+     if (settings->bools.video_ogs_vertical_enable)
+      rotation2 = HAL_TRANSFORM_ROT_180;
+
+     oga_blit(vid->msg_surface, 0, 0, vid->msg_width, vid->msg_height,
+        page_surface, 0, 0, vid->msg_width, vid->msg_height,
+        rotation2, vid->scale_mode, 0xff0105);
+    } else {
       oga_blit(vid->msg_surface, 0, 0, vid->msg_width, vid->msg_height,
             page_surface, 0, 0, vid->msg_height, vid->msg_width,
             HAL_TRANSFORM_ROT_270, vid->scale_mode, 0xff0105);
+    }
    }

    if (unlikely(drmModeSetCrtc(vid->fd, vid->crtc_id, page->fb_id, 0, 0, &vid->connector_id, 1, &vid->mode) != 0))
@@ -705,6 +733,14 @@ static void oga_set_rotation(void *data, unsigned rotation)
    if (!vid)
       return;

+   settings_t *settings                 = config_get_ptr();
+
+   if (settings->bools.video_oga_vertical_enable)
+   rotation = 3;
+
+   if (settings->bools.video_ogs_vertical_enable)
+   rotation = 2;
+
    switch (rotation)
    {
    case 0:

--- a/gfx/drivers_context/drm_go2_ctx.c
+++ b/gfx/drivers_context/drm_go2_ctx.c
@@ -295,7 +295,8 @@
    int src_w = drm->fb_width;
    int src_h = drm->fb_height;
    int src_x = 0;
    int src_y = drm->ctx_h - drm->fb_height;
+    settings_t *settings = config_get_ptr();

    if (out_w != src_w || out_h != src_h)
    {
@@ -321,10 +322,25 @@
    go2_context_swap_buffers(drm->context);
    surf     = go2_context_surface_lock(drm->context);
+
+if (settings->bools.video_oga_vertical_enable) {
+
+    auto ee_rotation = GO2_ROTATION_DEGREES_0;
+
+    if (settings->bools.video_ogs_vertical_enable)
+       ee_rotation = GO2_ROTATION_DEGREES_180;
+
+   go2_presenter_post(drm->presenter,
+         surf,
+         src_x, src_y, src_w, src_h,
+         out_y, out_x, out_h, out_w,
+         ee_rotation, 2);
+} else {
    go2_presenter_post(drm->presenter,
          surf,
          src_x, src_y, src_w, src_h,
          out_y, out_x, out_h, out_w,
          GO2_ROTATION_DEGREES_270, 2);
+}
    go2_context_surface_unlock(drm->context, surf);
 #endif
 }
