--- a/src/video/GLUtil.hh	2026-07-11 12:16:15.293086371 +0000
+++ b/src/video/GLUtil.hh	2026-07-11 12:17:19.623171532 +0000
@@ -22,7 +22,7 @@
 #define OPENGL_ES_2_0 1
 #define OPENGL_2_1    2
 #define OPENGL_3_3    3
-#define OPENGL_VERSION OPENGL_2_1
+#define OPENGL_VERSION OPENGL_ES_2_0
 
 namespace gl {
 
--- a/src/video/GLUtil.cc	2026-07-11 12:16:15.293019242 +0000
+++ b/src/video/GLUtil.cc	2026-07-11 12:17:19.637863803 +0000
@@ -148,18 +148,15 @@
 	if constexpr (OPENGL_VERSION == OPENGL_ES_2_0) {
 		source += "#version 100\n";
 		if (type == GL_FRAGMENT_SHADER) {
-			source += "#ifdef GL_FRAGMENT_PRECISION_HIGH\n"
-				"  precision highp float;\n"
-				"#else\n"
-				"  precision mediump float;\n"
-				"#endif\n";
+			
+				source += "precision highp float;\n";
 		}
 	} else {
 		source += "#version 110\n";
 	}
 	source += header;
 	try {
-		File file(systemFileContext().resolve(tmpStrCat("shaders/", filename)));
+		File file(preferSystemFileContext().resolve(tmpStrCat("shaders/", filename)));
 		auto mmap = file.mmap();
 		source.append(std::bit_cast<const char*>(mmap.data()),
 		              mmap.size());
--- a/src/3rdparty/imgui/imgui_impl_opengl3.cc	2026-07-11 12:16:15.169302546 +0000
+++ b/src/3rdparty/imgui/imgui_impl_opengl3.cc	2026-07-11 12:17:19.635563661 +0000
@@ -916,9 +916,9 @@
         "}\n";
 
     const GLchar* fragment_shader_glsl_120 =
-        "#ifdef GL_ES\n"
-        "    precision mediump float;\n"
-        "#endif\n"
+        
+        
+        
         "uniform sampler2D Texture;\n"
         "varying vec2 Frag_UV;\n"
         "varying vec4 Frag_Color;\n"
