--- a/src/video/scalers/GLScaler.cc	2026-07-11 12:16:15.300386100 +0000
+++ b/src/video/scalers/GLScaler.cc	2026-07-11 12:23:31.381542279 +0000
@@ -13,7 +13,7 @@
 GLScaler::GLScaler(const std::string& progName)
 {
 	for (auto i : xrange(2)) {
-		auto header = tmpStrCat("#define SUPERIMPOSE ", char('0' + i), '\n');
+		auto header = std::string();
 		VertexShader   vShader(header, progName + ".vert");
 		FragmentShader fShader(header, progName + ".frag");
 		program[i].attach(vShader);
